arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -486,7 +479,7 @@
         "tags": [
           "Operations"
         ],
-        "operationId": "getAvailableOperations",
+        "operationId": "Workspace_listOperations",
```

Comment:
Workspace_listOperations will cause a clash with Workspace resource (I think generated client will have some issue in Workspace, if not then this operation will end up in Workspace class meant for Workspace resource). Is there an issue you are fixing here?